Makefile

```diff
@@ -136,6 +142,18 @@ docker-build: test ## Build docker image with the manager.
 docker-push: ## Push docker image with the manager.
 	$(CONTAINER_TOOL) push ${IMG}
 
+GO_CONTAINER_CACHE = /tmp/dpu-operator-cache
+
+.PHONY: images-build
+images-build: test ## Build all container images
+	mkdir -p $(GO_CONTAINER_CACHE)
+	$(CONTAINER_TOOL) build -v $(GO_CONTAINER_CACHE):/go:z -f Dockerfile.rhel -t $(DPU_OPERATOR_IMAGE)
+	$(CONTAINER_TOOL) build -v $(GO_CONTAINER_CACHE):/go:z -f Dockerfile.daemon.rhel -t $(DPU_DAEMON_IMAGE)
+
+.PHONY: images-push
+images-push: test ## Build all container images
```

Comment:
Comment is not accurate.

You might want to add a small commentary on how to use this.

Create Local Registry.
make images-build
make images-push 
make local-deploy
Expect deployment to be created with DPU operator and DPU daemon

Reply:
Doesn't this build all images necessary to run the whole operator?

Reply:
@wizhaoredhat PTAL